src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -496,6 +497,7 @@ def load_arguments(self, _):
                    help="The eviction policy for virtual machines in a low priority scale set.", is_preview=True)
         c.argument('application_security_groups', resource_type=ResourceType.MGMT_COMPUTE, min_api='2018-06-01', nargs='+', options_list=['--asgs'], help='Space-separated list of existing application security groups to associate with the VM.', arg_group='Network', validator=validate_asg_names_or_ids)
         c.argument('computer_name_prefix', help='Computer name prefix for all of the virtual machines in the scale set. Computer name prefixes must be 1 to 15 characters long')
+        c.argument('empty', help='Create a VMSS without VM profile, which can be then referenced in `vm create`.', arg_type=get_three_state_flag())
```

Comment:
Nit: Creates 
Additional: If we change the term to orchestration mode, then all this description will change.

Reply:
When will you change?

Reply:
We have that term on portal already. Let's have a call today and clarify all the gaps?

Reply:
Decided on enum "orchestration" for this purpose. Keeping it in sync with Portal